frontend/package.json

```diff
@@ -27,16 +27,16 @@
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
     "eslint": "^5.16.0",
-    "jest": "^24.8.0",
+    "jest": "^25.1.0",
     "prop-types": "^15.7.2",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-hot-loader": "^4.11.0",
     "react-transform-catch-errors": "^1.0.2"
   },
   "peerDependencies": {
-    "@shopgate/pwa-common": "^6.7.1",
-    "@shopgate/pwa-common-commerce": "^6.7.1"
+    "@shopgate/pwa-common": "^6.11.0",
+    "@shopgate/pwa-common-commerce": "^6.11.0"
```

Comment:
we don't need 6.11 here for this extension, so we should not force it. since it might force some merchants to upgrade